src/main/java/com/github/cypher/sdk/Member.java

```diff
@@ -40,4 +40,9 @@ public int getPrivilege() {
 	IntegerProperty privilegeProperty() {
 		return privilege;
 	}
+
+	@Override
```

Comment:
If you override equals you should override hashCode too (otherwise HashMap etc wont function correctly).